### DISCUSSION TITLE
Ferramentas para criação de wireframes

### DISCUSSION THREAD

**Discussion by furabio** [frontendbr/forum#352] [Perguntas e Respostas]
Curso sistemas para internet e hoje na aula fui apresentado a diversas ferramentas de criação de wireframes, o que pra mim foi muito bom por não ser muito bom com photoshop e quando acabo tendo que criar uma layout lá, só sai merda rsrsrs, mas agora com essas ferramentas posso utilizar para desenhar minha layout e partir para o código(css, html, js).

Ai vem minha dúvida: 

Que ferramentas vocês costumam usar para a criação de seus wireframes, e vocês consideram uma boa prática cria-los antes de partir para o desenvolvimento da layout(código) ?

> Obs : fui apresentado ao moqups, AXURE(achei bem robusto), creately, Wireframe.cc e mockflow(que estou usando no momento).

**Comment by ghost** [comment]
Cara, ai depende. Se a ferramenta for ligada a um framework como o Bootstrap acho mais interessante pular essa etapa. Já testei vários programas WYSIWYG para Bootstrap e todos eles me pareceram dispensáveis.

Acho mais interessante criar um mock-up, uma  maquete funcional do site usando apenas o framework, sem pensar em cores, logo, nada. Isso, para mim pelo menos é mais eficiente do que fazer wireframes.

Geralmente os frameworks tem modelos de layout mais usados, também existem sites com snippets que tornam ainda mais supérfluo o wireframe. O drag-n-drop dos geradores geralmente acaba engessando todo o layout do site.

Concluindo, wireframes são algo ligado mais ao design que a programação. Eles são úteis como ferramenta de brainstorm, mas meio non-sense para a parte da programação em si.

**Comment by paulinhoerry** [comment]
Cara, eu usava o [Mavel](https://marvelapp.com/), achava ele super completo e elegante, até conhecer o [Invision](https://www.invisionapp.com/). O Invision é de outro mundo, de longe, a melhor ferramenta pra wireframe que eu já conheci.

**Comment by furabio** [comment]
@paulinhoerry depois vou testar essas duas que você citou :)

**Comment by casssiahaluu** [comment]
Já usei o bom e velho Pencil, e até que atendia. Então comecei a usar o [NinjaMock](https://ninjamock.com) e achei super prático

**Comment by felipebernardes** [comment]
Ei @furabio , eu gosto mais do Axure. Estudantes ganham licensa grátis! é só escanear a carteirinha de estudante. Você pode se inscrever [aqui](https://www.axure.com/edu)
